osf_tests/test_remind_draft_preregistrations.py

```diff
@@ -44,6 +44,23 @@ def test_dont_trigger_prereg_reminder_already_queued(self, draft):
 
         assert QueuedMail.objects.filter(email_type=PREREG_REMINDER_TYPE).count() == 1
 
+    def test_dont_trigger_if_node_deleted(self, draft):
```

Comment:
Minor: this test looks redundant with the other test you added.

Reply:
When I was writing this I thought I needed two tests here because we need `test_dont_trigger_if_node_deleted` to test that deleted draft emails never get added to the the queue and `test_dequeue_if_node_deleted` to ensure that emails already in the queue get dequeued even if they were initially queued correctly. So it looks redundant but you think about when `main` is run in prod it's important we check for both these things.